pkg/math/median/median.go

```diff
@@ -2,24 +2,19 @@ package median
 
 import (
 	"math/big"
-	"sort"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/skip-mev/slinky/oracle/types"
-	mmtypes "github.com/skip-mev/slinky/x/marketmap/types"
+	"github.com/skip-mev/slinky/pkg/math"
 )
 
-func ComputeMedianWithContext(_ sdk.Context) types.PriceAggregationFn {
-	return ComputeMedian()
-}
-
 // ComputeMedian inputs the aggregated prices from all providers and computes
 // the median price for each asset.
+//
+// NOTE: This function should only be used for testing purposes.
```

Comment:
If we use this in our main aggregation function now, we will effectively be calculating the medians per off-chain ticker and not the cannonical chain representation of the ticker.

Reply:
Let's move this to a test-utils